lib/mayaUsd/ufe/UsdMaterial.cpp

```diff
@@ -85,7 +106,7 @@ std::vector<Ufe::SceneItem::Ptr> UsdMaterial::getMaterials() const
     if (material) {
```

Comment:
You have the material right here. Add it to the results, and remove all the traversal of the `GetSurfaceOutputs()` since we no longer want to dig inside the material.